projects/rero/ng-core/src/lib/record/search/aggregation/aggregation.component.ts

```diff
@@ -42,6 +43,8 @@ export class RecordSearchAggregationComponent {
     return aggregationFilters === undefined ? [] : aggregationFilters.values;
   }
 
+  constructor(private _recordSearchService: RecordSearchService) {}
```

Comment:
missing (dummy) docstring.
But this service seems not used in this module. You could remove it (and remove import and constructor in the same).

Reply:
This has been removed.